src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java

```diff
@@ -33,4 +33,9 @@ public ResponseEntity<BaseResponse<String>> upload(@RequestParam("file") Multipa
             throw ErrorUtils.createApiError(HttpStatus.INTERNAL_SERVER_ERROR, HttpStatus.INTERNAL_SERVER_ERROR.getReasonPhrase(), "Some file error occurred", e.getStackTrace());
         }
     }
+
+    private void CheckFileSanity(MultipartFile file) throws Exception {
```

Comment:
Please make it start with small case letter

Reply:
There is no harm in this format also for private methods.

Reply:
Please check this [link](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf ). 
<img width="687" alt="image" src="https://user-images.githubusercontent.com/32239582/215325433-71da2acc-996e-4863-a32d-01ab45b802c0.png">

Reply:
Read the conventions on the net also but couldnt find a reason for it. FYI already modified it in the next PR.